operations/configuration_management/View_Configuration_Session_Logs.md

```diff
@@ -1,18 +1,18 @@
 # View Configuration Session Logs
 
-Logs for the individual steps of a session are available via the `kubectl log` command for each container of a Configuration Framework Service \(CFS\) session. Refer to [Configuration Sessions](Configuration_Sessions.md) for more info about these containers.
+Logs for the individual steps of a session are available via the `kubectl log` command for each container of a Configuration Framework Service \(CFS\) session. Refer to
+[Configuration Sessions](Configuration_Sessions.md) for more info about these containers.
 
 To find the name of the Kubernetes pod that is running the CFS session:
 
 ```bash
-ncn# kubectl get pods --no-headers -o \
-custom-columns=":metadata.name" -n services -l cfsession=example
+ncn# kubectl get pods --no-headers -o custom-columns=":metadata.name" -n services -l cfsession=example
 ```
 
 Store the returned pod name as the `CFS_POD_NAME` variable for future use:
 
 ```bash
-ncn# export CFS_POD_NAME=cfs-f9d18751-e6d1-4326-bf76-434293a7b1c5-q8tsc
+ncn# CFS_POD_NAME=cfs-f9d18751-e6d1-4326-bf76-434293a7b1c5-q8tsc
```

Comment:
ncn# doesn't seem right. I don't think this will be something you run from a storage node (only k8s access on s001

Reply:
Addressing this with https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-4815